frontend/src/components/Arrivals.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect, useRef } from 'react';
+import './Arrivals.css';
+
+const Arrivals = ({ tubeData, durationPassed }) => {
+  const [newArrival, setNewArrival] = useState('');
+  const durationPassedRef = useRef(durationPassed);
+  
+  useEffect(() => {
+    
+    const scheduleTrains = () => {
+      const trainData = tubeData;
+      const arrivingNext = [];
```

Comment:
Am I right in thinking arrivingNext is an array because it could contain several different instantaneous arrivals on different lines?

Reply:
yes indeedy!